### ISSUE TITLE
Update DSF minutes to include "apologies" as well as Attendees

### ISSUE THREAD

**Issue by nanorepublica** [django/djangoproject.com#1704]
It's common in meeting minutes to include a list of those not able to attend as well as those in attendance. 

Therefore I would like to propose this gets added to the DSF meeting minute pages.

**Comment by bmispelon** [comment]
This doesn't seem too complicated to do on a technical level (should be a straightforward addition to `foundation.models.Meeting` and a few template edits), but does that change need to be approved first (possibly by the DSF board)?

**Comment by nanorepublica** [comment]
The same applies here:
https://github.com/django/djangoproject.com/issues/1705#issuecomment-2443933987

**Comment by thibaudcolas** [comment]
I think this would be great for transparency, I’ll check with other board members as I’m surprised we don’t already have this in place considering we do already keep track of this in our internal notes.

**Comment by thibaudcolas** [comment]
Feedback from board people is this is good to go.

**Comment by thibaudcolas** [comment]
I’ve un-assigned myself as I don’t think I’ll be able to pick this up in the end. Here is the data for 2024 meetings if anyone wants to pick this up and is interested to try this out with real-world content. Once this is in place, please create a new issue with me assigned for me to add content for 2023 and any older minutes.

<details>

<summary>2024 DSF Board meetings attendance</summary>

## December 10th 2024

| **Member**            | **Attending?**           |
| --------------------- | ------------------------ |
| Chaim                 | ✅ (Chair)               |
| Çağıl                 | ✅                       |
| Jacob                 | ✅                       |
| Thibaud               | ✅                       |
| Kátia                 | ✅                       |
| Sarah                 | ✅                       |
| Katie                 | ✅                       |
| **Non-board members** |                          |
| Catherine             | ✅                       |
| Incoming Board:       | ✅ Afi, Jeff, Paolo, Tom |

## November 19th, 2024

| **Member**            | **Attending?** |
| --------------------- | -------------- |
| Chaim                 | ✅ (Chair)     |
| Çağıl                 | ✅             |
| Jacob                 | ✅             |
| Thibaud               | ✅             |
| Kátia                 | ✅             |
| Sarah                 | ✅             |
| Katie                 | ✅             |
| **Non-board members** |                |
| Catherine             | ✅             |

## October 10 2024

| **Member**            | **Attending?** |
| --------------------- | -------------- |
| Chaim                 | ✅ (Chair)     |
| Çağıl                 | ✅             |
| Jacob                 | ❌             |
| Thibaud               | ✅             |
| Kátia                 | ✅             |
| Sarah                 | ✅             |
| Katie                 | ✅             |
| **Non-board members** |                |
| Catherine             | ✅             |

## September 12 2024

| **Member**            | **Attending?** |
| --------------------- | -------------- |
| Chaim                 | ✅ (Chair)     |
| Çağıl                 | ❌             |
| Jacob                 | ❌             |
| Thibaud               | ✅             |
| Kátia                 | ✅             |
| Sarah                 | ✅             |
| Katie                 | ✅             |
| **Non-board members** |                |
| Catherine             | ✅             |

## August 8 2024

| **Member**            | **Attending?** |
| --------------------- | -------------- |
| Chaim                 | ✅ (Chair)     |
| Çağıl                 | ✅             |
| Jacob                 | ✅             |
| Thibaud               | ✅             |
| Kátia                 | ✅             |
| Sarah                 | ✅             |
| Katie                 | ✅             |
| **Non-board members** |                |
| Catherine             | ✅             |

## July 18 2024

| **Member**            | **Attending?** |
| --------------------- | -------------- |
| Chaim                 | ✅ (Chair)     |
| Çağıl                 | ✅             |
| Jacob                 | ❌             |
| Thibaud               | ✅             |
| Kátia                 | ✅             |
| Sarah                 | ✅             |
| Katie                 | ❌             |
| **Non-board members** |                |
| Catherine             | ✅             |

## June 13th 2024

| **Member**            | **Attending?** |
| --------------------- | -------------- |
| Chaim                 | ❌             |
| Çağıl                 | ✅ (Chair)     |
| Jacob                 | ✅             |
| Thibaud               | ✅             |
| Kátia                 | ✅             |
| Sarah                 | ✅             |
| Katie                 | ✅             |
| **Non-board members** |                |
| Catherine             | ✅             |

## June 6th 2024

| **Member**            | **Attending?** |
| --------------------- | -------------- |
| Chaim                 | ✅             |
| Çağıl                 | ✅ (Chair)     |
| Jacob                 | ✅             |
| Thibaud               | ✅             |
| Kátia                 | ✅             |
| Sarah                 | ✅             |
| Katie                 | ❌             |
| **Non-board members** |                |
| Catherine             | ✅             |

## May 16th 2024 (postponed from May 9

| **Member**            | **Attending?** |
| --------------------- | -------------- |
| Chaim                 | ✅ (Chair)     |
| Çağıl                 | ✅             |
| Jacob                 | ✅             |
| Thibaud               | ✅             |
| Kátia                 | ❌             |
| Sarah                 | ✅             |
| Katie                 | ✅             |
| **Non-board members** |                |
| Catherine             | ✅             |

## April 11th 2024

| **Member**            | **Attending?** |
| --------------------- | -------------- |
| Chaim                 | ✅ (Chair)     |
| Çağıl                 | ❌             |
| Jacob                 | ✅             |
| Thibaud               | ✅             |
| Kátia                 | ✅             |
| Sarah                 | ✅             |
| Katie                 | ❌             |
| **Non-board members** |                |
| Catherine             | ✅             |

## March 14th 2024

| **Member**            | **Attending?**    |
| --------------------- | ----------------- |
| Chaim                 | ❌                |
| Çağıl                 | ✅                |
| Jacob                 | ✅                |
| Thibaud               | ✅                |
| Kátia                 | ❌                |
| Sarah                 | ❌                |
| Katie                 | ✅ (meeting lead) |
| **Non-board members** |                   |
| Catherine             | ✅                |

## February 8th 2024

| **Member**            | **Attending?** |
| --------------------- | -------------- |
| Chaim                 | ✅ (Chair)     |
| Çağıl                 | ✅             |
| Jacob                 | ✅             |
| Thibaud               | ✅             |
| Kátia                 | ✅             |
| Sarah                 | ✅             |
| Katie                 | ✅             |
| **Non-board members** |                |
| Catherine             | ✅             |

## January 11th 2024

| **Member**            | **Attending?** |
| --------------------- | -------------- |
| Chaim                 | ✅ (Chair)     |
| Çağıl                 | ✅             |
| Jacob                 | ✅             |
| Thibaud               | ✅             |
| Kátia                 | ✅             |
| Sarah                 | ✅             |
| Katie                 | ✅             |
| **Non-board members** |                |
| Catherine             | ✅             |

</details>

**Comment by thibaudcolas** [comment]
cc our 2025 Secretary, @knyghty

**Comment by sarahboyce** [comment]
Given the DSF meeting minutes now live in https://github.com/django/dsf-minutes (see https://github.com/django/djangoproject.com/issues/2079). I suppose we can close this PR and this feedback can be incorporated in the DSF secretary monute writing process (cc @knyghty)

**Comment by knyghty** [comment]
I'm not the secretary anymore but I think this is reasonable.

@priyapahwa for context, maybe good to add this into the minutes template for this year.

**Comment by priyapahwa** [comment]
Noted. Thanks for highlighting this.
